src/modules/Blend/info.json

```diff
@@ -10,7 +10,7 @@
     "blend": {
       "type": "string",
       "desc": "Function to use to blend the two images.",
-      "default": "function(r1, g1, b1, a1, r2, g2, b2, a2) { return [ r1, g2, b2, a2 ] }"
+      "default": "function(r1, g1, b1, a1, r2, g2, b2, a2,x,y) { return [ r2, g2, b2, a2 ] }"
```

Comment:
Here too, please add spaces after the last two parameters because there are spaces in the first few.